tests/integration/test_causal_binary_classification_modeling_cfg.yaml

```diff
@@ -12,7 +12,8 @@ dataset:
     add_eos_token_to_answer: false
     add_eos_token_to_prompt: false
     add_eos_token_to_system: false
-    answer_column: binary_label
+    answer_column: 
+    - binary_label
```

Comment:
This required change is a bit unfortunate as the error message is rather cryptic for CLI users when using previously well working yamls.

```
Traceback (most recent call last):
  File "/home/xxx/h2o-llmstudio/train.py", line 722, in <module>
    run(cfg=cfg)
  File "/home/xxx/h2o-llmstudio/train.py", line 530, in run
    train_dataset = get_train_dataset(train_df=train_df, cfg=cfg)
  File "/home/xxx/h2o-llmstudio/llm_studio/src/utils/data_utils.py", line 396, in get_train_dataset
    train_dataset: Dataset = cfg.dataset.dataset_class(
  File "/home/xxx/h2o-llmstudio/llm_studio/src/datasets/text_causal_classification_ds.py", line 19, in __init__
    check_for_non_int_answers(cfg, df)
  File "/home/xxx/h2o-llmstudio/llm_studio/src/datasets/text_causal_classification_ds.py", line 104, in check_for_non_int_answers
    x for x in df[column].values if not is_castable_to_int(x)
  File "/home/xxx/.local/share/virtualenvs/h2o-llmstudio-tT3gHl3a/lib/python3.10/site-packages/pandas/core/frame.py", line 4102, in __getitem__
    indexer = self.columns.get_loc(key)
  File "/home/xxx/.local/share/virtualenvs/h2o-llmstudio-tT3gHl3a/lib/python3.10/site-packages/pandas/core/indexes/base.py", line 3812, in get_loc
    raise KeyError(key) from err
KeyError: 'w'
```